src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -176,6 +176,344 @@ namespace cryptonote
+          KV_SERIALIZE(coinbase)
+          KV_SERIALIZE(mempool)
+          KV_SERIALIZE(mixin)
+        END_KV_SERIALIZE_MAP()
+      };
+      
+      
+      struct response
+      {
+        //std::list<std::string> txs_as_json;
+        std::string total_received;
+        uint64_t scanned_height;
+        std::list<transaction> transactions;
+        uint64_t blockchain_height;
+        uint64_t scanned_block_height;
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(total_received)
+          KV_SERIALIZE(scanned_height)
+          KV_SERIALIZE(transactions)
+          KV_SERIALIZE(blockchain_height)
+          KV_SERIALIZE(scanned_block_height)
+        END_KV_SERIALIZE_MAP()
+      };
+  };
+  
+  //-----------------------------------------------
+  struct COMMAND_RPC_LIGHT_WALLET_GET_ADDRESS_INFO
+  {
+      struct request
+      {
+        std::string address;
+        std::string view_key;
+
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(address)
+          KV_SERIALIZE(view_key)
+        END_KV_SERIALIZE_MAP()
+      };
+      
+      struct spent_output 
+      {
+        std::string amount;
+        std::string key_image;
+        std::string tx_pub_key;
+        int out_index;
+        int mixin;
+        
+        
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(amount)
+          KV_SERIALIZE(key_image)
+          KV_SERIALIZE(tx_pub_key)
+          KV_SERIALIZE(out_index)
+          KV_SERIALIZE(mixin)
+        END_KV_SERIALIZE_MAP()
+      };
+  
+      
+      
+      struct response
+      { 
+        std::string locked_funds;
+        std::string total_received;
+        std::string total_sent;
+        uint64_t scanned_height;
+        uint64_t scanned_block_height;
+        uint64_t start_height;
+        uint64_t transaction_height;
+        uint64_t blockchain_height;
+        std::list<spent_output> spent_outputs;
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(locked_funds)
+          KV_SERIALIZE(total_received)
+          KV_SERIALIZE(total_sent)        
+          KV_SERIALIZE(scanned_height)
+          KV_SERIALIZE(scanned_block_height)
+          KV_SERIALIZE(start_height)
+          KV_SERIALIZE(transaction_height)
+          KV_SERIALIZE(blockchain_height)
+          KV_SERIALIZE(spent_outputs)
+        END_KV_SERIALIZE_MAP()
+      };
+  };
+  
+  //-----------------------------------------------
+  struct COMMAND_RPC_LIGHT_WALLET_GET_UNSPENT_OUTS
+  {
+      struct request
+      {
+        std::string amount;
+        std::string address;
+        std::string view_key;
+
+        BEGIN_KV_SERIALIZE_MAP()
+          KV_SERIALIZE(amount)
+          KV_SERIALIZE(address)
+          KV_SERIALIZE(view_key)
+        END_KV_SERIALIZE_MAP()
+      };
+    
+      
+      struct output {
+        std::string amount;
+        std::string public_key;
+        int index;
+        uint64_t global_index;
+        std::string rct;
+        uint64_t tx_id;
```

Comment:
Maybe @vtnerd or @fluffypony could shed some light here? Here's example output from mymonero on the get_unspent_outs api call, as you can see it's not a hash. I'm not using it the wallet code so i'm suspecting it's an internal MyMonero id that could be dropped? Whatever it is, i agree that the name "tx_id" key shouldn't be used. =)

```
 "per_kb_fee": 286360000,
  "amount": "3470620238100",
  "outputs": [
    {
      "amount": "500000000000",
      "public_key": "1976ee95fe9751d461897ab53a6331b89461195a724342f3d7235bc4630a053b",
      "index": 0,
      "global_index": 797417,
      "rct": "c22e7dd61b95861413ecc4c4cde586ba716dc1a0d2a07b08be0a160a0cd454c1bae7db5bed19173565b30ab89f0c2425c10dcf2f56111a20e83904bb4fcb620a85013a82c860990508bc8970b2a3064375505e28a584d4911a3cca3295699e07",
      "tx_id": 2507349,
      "tx_hash": "b3bd2cc28c49b094ed29bac1f5386a300e7286e8bf9060dce6887c4ac1247a68",
      "tx_pub_key": "20bf3f1deda1c21db37e87652a9505b6617c2cc94b162583ef367da22be8f5c7",
      "tx_prefix_hash": "c39a83988b6e0c04bf889b7f1c8989a5467d3c2a5383f50cc0685ff7fcff93de",
      "spend_key_images": [
        "e04139a08a607bed4c455f7b19f91d8272e0aaa2b53ead740430bde691a8b6e6",
        "987ece6e5cc12c7734e4317a658ac5fbdca9f66f190b8af9965a731beeaa12e1",
        "69b87febb6019c938e5e2861bc54d66605927f07861ab533671d4edb4f7a7007",
        "74856312f49ca618af3b66a33b28fa557d30b80b764dd57f19276284758c3281"
      ],
      "timestamp": "2017-06-27T12:06:58Z",
      "height": 1304290
    }, 
...
```